templates/staff/workspace/detail.html

```diff
@@ -94,7 +94,7 @@
             {% fragment as title %}
               {{ job_request.pk }} by {{ job_request.created_by.name }}
             {% endfragment %}
-            {% #list_group_rich_item status_text=job_request.status title=title type="Job" url=job.get_absolute_url %}
+            {% #list_group_rich_item status_text=job_request.status title=title type="Job" url=job_request.get_absolute_url %}
```

Comment:
Typically, within the staff area we link to other staff area pages, so I think this link should be `job_request.get_staff_url` instead.

Reply:
Ah, thanks; I'd not clicked around enough in the staff area to be aware there was a distinction.